pkg/tsdb/tempo/tempo_test.go

```diff
@@ -10,9 +10,16 @@ import (
 )
 
 func TestTempo(t *testing.T) {
-	t.Run("createRequest - success", func(t *testing.T) {
+	t.Run("createRequest without time range - success", func(t *testing.T) {
 		service := &Service{tlog: log.New("tempo-test")}
-		req, err := service.createRequest(context.Background(), &datasourceInfo{}, "traceID")
+		req, err := service.createRequest(context.Background(), &datasourceInfo{}, "traceID", 0, 0)
+		require.NoError(t, err)
+		assert.Equal(t, 1, len(req.Header))
+	})
+
+	t.Run("createRequest with time range - success", func(t *testing.T) {
+		service := &Service{tlog: log.New("tempo-test")}
+		req, err := service.createRequest(context.Background(), &datasourceInfo{}, "traceID", 1, 2)
 		require.NoError(t, err)
 		assert.Equal(t, 1, len(req.Header))
```

Comment:
Would be nice to also assert that the query is formatted correctly in this test case